src/main/java/edu/nextstep/camp/calc/domain/PositiveNumber.java

```diff
@@ -0,0 +1,72 @@
+package edu.nextstep.camp.calc.domain;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+// Note: 엄밀히 따지면 0을 포함하므로 NonNegativeNumber가 맞지만, 이름이 너무 길고 부정적이라 이렇게 함
+// Note: 대안으로는 UnsignedNumber나 AbsoluteNumber도 가능은 하지만, 용도와 맞지 않는다고 판단.
+public class PositiveNumber {
+    private final static Pattern NUMBER_PATTERN = Pattern.compile("[0-9]+");
+    private final static String TO_STRING_FORMAT = "Number(%d)";
+    private static final int ZERO_VALUE = 0;
+    private static final PositiveNumber ZERO = new PositiveNumber(ZERO_VALUE);
```

Comment:
`private static final` 이 맞습니다 :) 

```suggestion
    private static final Pattern NUMBER_PATTERN = Pattern.compile("[0-9]+");
    private static final String TO_STRING_FORMAT = "Number(%d)";
    private static final int ZERO_VALUE = 0;
    private static final PositiveNumber ZERO = new PositiveNumber(ZERO_VALUE);
```

Reply:
ZERO는 public으로 열어도 괜찮을 것 같네요 :)

Reply:
private static final int ZERO_VALUE = 0;
0은 상수로 뺄 필요가 있을까요? 상수 이름 또한 0인데 불필요한 것 같네요 :)

Reply:
제가 계속 static과 final의 순서를 바꿔가며 썼네요 ㅠㅠ 앞으로 신경 쓰겠습니다.

ZERO_VALUE의 경우는
원래 ZERO를 만들 때는 new PositiveNumber(0);이라고 썻었고..
0이상을 확인하기 위한 POSITIVE_THRESHOLD가 있었습니다. 값은 당연히 0이었구요.

리팩토링을 하다보니 PositiveNumber.of(0)도 ZERO를 리턴하기 위해 값이 0인지를 검사하다 보니
다른 의미이지만 반복적으로 사용되는것 같아 이름을 ZERO_VALUE라고 고쳤는데요
마지막까지 고민이 되긴했었습니다. ㅠㅠ